flake.nix

```diff
@@ -103,7 +103,7 @@
           default = pkgs.mkShell {
             packages = with pkgs; [
               (fenixToolchain system)
-              cargo-watch
+              bacon
```

Comment:
I rather like cargo watch?

Reply:
I promise you'll like bacon more. But I can keep them both.